### DISCUSSION TITLE
[Chapter 11] 한글 문서화 주석 작성 시 적절한 문체

### DISCUSSION THREAD

**Discussion by corock** [cowaine/is-my-code-that-strange#102] [Chapter 11-12 (주석, 메서드)] [answered]
p. 258 코드 11.7 에서 _Javadoc_ 형식으로 작성된 코드 스니펫이 나옵니다.

```java
class Money {

    // (...)

    /**
     * 금액을 추가합니다.
     *
     * @param other 추가 금액
     * @return 추가 후의 금액
     * @throws IllegalArgumentException 통화 단위가 다르면 예외 발생
     */
    Money add(final Money other) {
        // (...)
    }

}
```

여기에서 '금액을 추가합니다.' 라는 문장은 우리나라 문체 중 [경어체](https://encykorea.aks.ac.kr/Article/E0002676)에 속합니다.
반면, <이펙티브 자바> 등의 도서에서 문서화 주석이 달린 코드 스니펫은 모두 [평어체](https://www.joongang.co.kr/article/25130505)를 사용합니다.

가령 어떤 개발팀에서 문서화 주석을 작성하자는 컨벤션이 있을 때, 팀원 A 는 경어체를 쓰는 반면 팀원 B 는 평어체를 쓰는 상황이 생길 것 같은데요.
영어는 이런 점을 크게 고려할 필요가 없는 편인데, 위 사례가 생기면 어떻게 대처하실 건가요? (정답이 없을 것 같지만 선호하는 방식이 궁금합니다.)

**Comment by firewoody237** [comment, marked as answer]
기존 doc들이 경어체로 적혀있다면 경어체로 적을 것 같습니다.

내부 API서버이고, 제가 처음으로 작성해야 한다면 평어체로 적을 것 같습니다.
다만 대고객용(외부 제공 API)와 같은경우엔 조금 고민할 것 같네요..
